File: rest-demo/app/src/main/java/momatechsoftwares/rest_demo/networking/services/MovieService.java
package momatechsoftwares.rest_demo.networking.services;

import android.support.annotation.NonNull;

import java.util.List;

import momatechsoftwares.rest_demo.networking.Routes;
import momatechsoftwares.rest_demo.networking.models.Movie;
import momatechsoftwares.rest_demo.networking.response.AuthResponse;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface MovieService {

    @NonNull
    @GET(Routes.TOP_RATED_ENDPOINT)
    Call<AuthResponse> getTopRatedMovies(@Query("api_key") String apiKey);
}

File: bookit.ng/app/src/main/java/com/bookitng/bookitng/networking/services/MovieService.kt
package com.bookitng.bookitng.networking.services

import com.bookitng.bookitng.models.TodayMovies
import com.bookitng.bookitng.networking.Routes
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Header
import retrofit2.http.Query

interface MovieService {

    @GET(Routes.TODAY_MOVIES_ENDPOINT)
    fun retrieveTodayMovies(@Header("Authorization") bearerToken: String?,
                            @Query("limit") limit: Int,
                            @Query("page") page: Int): Call<TodayMovies>
}